sandpack-react/src/types.ts

```diff
@@ -241,6 +241,7 @@ export type SandpackPredefinedTemplate =
   | "vanilla-ts"
   | "vue"
   | "vue3"
+  | "vue3-ts"
```

Comment:
I was wondering if we could make this smarter instead of a manual list of types?
```
export type SandpackPredefinedTemplate = keyof typeof SANDBOX_TEMPLATES;
 ```

What do you think?

Reply:
👍🏻
I have updated.